docs/website/docs/ml-frameworks/tensorflow-lite.md

```diff
@@ -1,14 +1,61 @@
 # TensorFlow Lite Integration
 
-!!! todo
-    [Issue#5455](https://github.com/google/iree/issues/5455): write this documentation
+IREE supports compiling and running pre-trained TFLite models.  It converts a model to [TOSA MLIR](https://mlir.llvm.org/docs/Dialects/TOSA/), then compiles it into a VM module.
```

Comment:
I think we should spell out "TensorFlow Lite" throughout, or at least explain the abbreviation (`pre-trained TensorFlow Lite (TFLite) models`).

```suggestion
IREE supports compiling and running pre-trained TensorFlow Lite models. 
It converts a model to [TOSA MLIR](https://mlir.llvm.org/docs/Dialects/TOSA/),
then compiles it into a VM module.
```